lib/modules/hosts/youtube.js

```diff
@@ -8,7 +8,7 @@ modules['showImages'].siteModules['youtube'] = {
 	handleLink: function(elem) {
 		var def = $.Deferred();
 
-		var hashRe = /^https?:\/\/(?:www\.|m\.)?youtube\.com\/watch\?.*v=([\w\-]+)/i;
+		var hashRe = /^https?:\/\/(?:www\.|m\.)?youtube\.com\/watch.*?[?&]v=([\w\-]+)/i;
```

Comment:
Think we can narrow down the highlighted part a bit more, maybe with `\/watch\?(?:[^?&]+&)*?v=([\w\-]+)`. I don't know how valuable that would be though.